e2e/src/step_definitions/media-viewer.ts

```diff
@@ -127,6 +127,32 @@ Then('I expect to see the document should be downloaded', async function () {
   await downloadPage.waitForDownloadToComplete(file);
 });
 
+When('the user selects the multimedia option', async () => {
+  await multimediaPage.clickMultimedia();
+  await genericMethods.sleep(5000);
+});
+
+When('the user selects play option', async () => {
+  await multimediaPage.clickPlayButton();
+});
+
+Then('I should see video should be in play mode', async function ()  {
```

Comment:
if we're just taking a screenshot this becomes a manual test rather than automated
is there anything in the markup we can check to see if the video is playing?
from what I could see, the play button isn't clickable if the video can't be played

Reply:
Let me check...

Reply:
Don't find the solution. That's the optimistic solution at the moment and we can improve it later